spec/spark_worker_spec.rb

```diff
@@ -43,6 +53,15 @@
       expect(chef_run).to run_ruby_block("package-#{pkg}")
     end
 
+    %W(
```

Comment:
Yes and no. It's the same check, but a different context. The check is redundant, though, since the resources aren't context-sensitive. I'll clean it up in my current branch, if that's alright.